include/SFML/System/Time.hpp

```diff
@@ -79,6 +87,14 @@ class Time
     ////////////////////////////////////////////////////////////
     constexpr Int64 asMicroseconds() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Return the time value as a std::chorono::duration
+    ///
+    /// \return Time in microseconds
+    ///
+    ////////////////////////////////////////////////////////////
+    constexpr std::chrono::microseconds toDuration() const;
```

Comment:
Wouldn't it be more consistent to name this method `asDuration`? to be more like `asSeconds` etc.

Reply:
I thought about that. I'm not entirely sure. We're using `to` in `sf::Color::toInteger` for what it's worth.

Reply:
On second thought, maybe it is ok not to be consistent as conversion between units and types is not exactly the same idea.